src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: Avenir-Heavy, serif;
+}
+
+.body {
+  margin: 0 8px;
+}
+
+.form-big {
+  margin: 20px 0;
+}
+
+.search-big {
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  background: url("./images/Search.svg") 25px 25px / 19px 19px no-repeat;
+
+  border-radius: 4px;
+  font-size: 16px;
+  line-height: 70px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.search-big::placeholder {
+  font-family: Avenir, serif;
+}
+
+.search-small {
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  box-sizing: border-box;
+  background: url("./images/Search.svg") 13px 15px / 11px 11px no-repeat;
+  border-radius: 4px;
+  font-size: 14px;
+  line-height: 42px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 300;
+  font-family: inherit;
+}
+
+.search-small::placeholder {
+  font-family: Avenir, serif;
+}
+
+.search-big > input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
```

Comment:
Do not increase specificity, use class selector

Reply:
fixed it